mockgcp/generated/mockgcp/cloud/gkehub/v1beta/feature.pb.go

```diff
@@ -635,6 +637,13 @@ func (x *MembershipFeatureSpec) GetMesh() *servicemeshpb.MembershipSpec {
 	return nil
 }
 
+func (x *MembershipFeatureSpec) GetPolicycontroller() *policycontrollerpb.MembershipSpec {
+	if x, ok := x.GetFeatureSpec().(*MembershipFeatureSpec_Policycontroller); ok {
+		return x.Policycontroller
+	}
+	return nil
+}
+
```

Comment:
interesting so the mock generation added these lines when adding the policy controller pb ?

Reply:
Yes, I made the upstream change in google apis in b/376542425